packages/types/src/codec/BTreeMap.ts

```diff
@@ -0,0 +1,183 @@
+import { Compact, U8a } from '.';
```

Comment:
```suggestion
import Compact from './Compact';
import U8a from './U8a';
```

Rationale for sticking to single imports - the `BTreeMap` will be in `index.ts` as well and we want to be explicit to avoid any chance of circular deps. (We have some issues in this regard)